src/content/configuration/dev-server.md

```diff
@@ -1353,6 +1353,108 @@ webpack-dev-server --stdin
 ```
 
 
+## `devServer.transportMode`
+
+`string: 'sockjs' | 'ws'` `object`
```

Comment:
The default value is object, lets move it to the first place and mention the default
```suggestion
`object: { client: 'sockjs',  server: 'sockjs' }` `string: 'sockjs' | 'ws'`
```

Reply:
I think that it makes more sense to say that the default is 'sockjs', even if it is internally that equivalent object. Most basic users I suspect would only make use of the string options.

Reply:
Maybe something like: `string: 'sockjs' | 'ws'` `object: { client: ... ,  server: ... }`

and then elaborate on the object properties later?

Reply:
okay as long as we say that setting `devServer.transportMode` to a string is a shortcut to setting both `devServer.transportMode.client | server` to that value